x-pack/plugins/security_solution/public/management/components/console/components/command_list.test.tsx

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ConsoleTestSetup, HelpSidePanelSelectorsAndActions } from '../mocks';
+import {
+  getCommandListMock,
+  getConsoleTestSetup,
+  getHelpSidePanelSelectorsAndActionsMock,
+} from '../mocks';
+import React from 'react';
+import { waitFor } from '@testing-library/react';
+
+describe('When rendering the command list (help output)', () => {
+  let render: ConsoleTestSetup['renderConsole'];
+  let renderResult: ReturnType<typeof render>;
+  let consoleSelectors: ConsoleTestSetup['selectors'];
+  let enterCommand: ConsoleTestSetup['enterCommand'];
+
+  beforeEach(() => {
+    const testSetup = getConsoleTestSetup();
+
+    render = (props = {}) => (renderResult = testSetup.renderConsole(props));
+    consoleSelectors = testSetup.selectors;
+    enterCommand = testSetup.enterCommand;
+  });
+
+  describe('and its displayed on the side panel', () => {
+    let renderAndOpenHelpPanel: typeof render;
+    let helpPanelSelectors: HelpSidePanelSelectorsAndActions;
+
+    beforeEach(() => {
+      renderAndOpenHelpPanel = (props) => {
+        render(props);
+        helpPanelSelectors = getHelpSidePanelSelectorsAndActionsMock(renderResult);
+        consoleSelectors.openHelpPanel();
+
+        return renderResult;
+      };
+    });
+
+    it('should display the help panel header', () => {
+      renderAndOpenHelpPanel();
+
+      expect(renderResult.getByTestId('test-sidePanel-header')).toHaveTextContent(
+        'HelpUse the add () button to populate a response action to the text bar. Add ' +
+          'additional parameters or comments as necessary.'
+      );
+    });
+
+    it('should display the command list', () => {
+      renderAndOpenHelpPanel();
+
+      expect(renderResult.getByTestId('test-commandList')).toBeTruthy();
```

Comment:
I think that we do not need `.toBeTruthy()` or any kind of `.exists()` at the end when looking for an element.
Isn't getByTest Id the assertion by itself? If it doesn't find it - the test fails ;) 

What do you think about it?

Reply:
yeah, true. but does not hurt, and (for me) its clearer to understand that the assertion is attempting to validate.